Python/legacy_tracing.c

```diff
@@ -59,6 +59,16 @@ static PyObject *
 sys_profile_func3(
```

Comment:
I think this is now only used for throw events, so maybe rename it to `sys_profile_throw`?

Reply:
I always think the callbacks should have better names. I thought there are some naming conventions used here. I changed names of all the single-used callbacks to involve the event they are used for.